test: go test failing locally

To onboard new maintainers, it would be better if `go test ./...` worked out of the box.

right now it seems it requests some external setup.

also would be good to introduce `-short` support for unit testing only.


<details>
<summary>...</summary>
<pre>
?   	github.com/sigstore/cosign/cmd/cosign	[no test files]
?   	github.com/sigstore/cosign/cmd/cosign/cli	[no test files]
?   	github.com/sigstore/cosign/copasetic	[no test files]
=== RUN   TestLoadPrivateKey
--- PASS: TestLoadPrivateKey (0.42s)
PASS
ok  	github.com/sigstore/cosign/pkg/cosign	(cached)
?   	github.com/sigstore/cosign/pkg/cosign/kms	[no test files]
=== RUN   TestSignVerify
    e2e_test.go:353: COSIGN_TEST_REPO unset, using fake registry
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:c8fa724cb05b9c99afa8ef70d3b7da5f9622613cf736f013fadaae5077842a58 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:e28066df07ecde0d32905c66fc35b26804b5cf3125fa6116aac4e88a7040b69e 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:3fe5fdce124718213c3dece6f71632fad0450ced88228501e3077fbca1b1b840 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:801e45da5d42fffbe2910687b444d877bb20a15efbfa7ad502974aa2c4a20bf6 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:6327342866a29c3bbfe5c32a69111e4d6912a1321c3a5ff25cdeea48819a37aa 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:9496397081e5318659b1e728d9b5ba7d3167c68262bfa2b7c17d1788fe4daaec 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/605394647632969758
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/1443635317331776148
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/6334824724549167320
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/2775422040480279449
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/894385949183117216
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/1443635317331776148?digest=sha256%3A3fe5fdce124718213c3dece6f71632fad0450ced88228501e3077fbca1b1b840
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/605394647632969758?digest=sha256%3Ae28066df07ecde0d32905c66fc35b26804b5cf3125fa6116aac4e88a7040b69e
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/4751997750760398084
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/6334824724549167320?digest=sha256%3Ac8fa724cb05b9c99afa8ef70d3b7da5f9622613cf736f013fadaae5077842a58
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/2775422040480279449?digest=sha256%3A801e45da5d42fffbe2910687b444d877bb20a15efbfa7ad502974aa2c4a20bf6
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/894385949183117216?digest=sha256%3A9496397081e5318659b1e728d9b5ba7d3167c68262bfa2b7c17d1788fe4daaec
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/4751997750760398084?digest=sha256%3A6327342866a29c3bbfe5c32a69111e4d6912a1321c3a5ff25cdeea48819a37aa
2021/03/11 11:42:25 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
Pushing signature to: 127.0.0.1:63572/cosign-e2e:sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:a32319b5ad16e9cf1501850ca2fbc90f4dc23341123872bb8b1fe50f1dfc6226 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:d1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/7504504064263669287
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/1976235410884491574
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/7504504064263669287?digest=sha256%3Aa32319b5ad16e9cf1501850ca2fbc90f4dc23341123872bb8b1fe50f1dfc6226
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/1976235410884491574?digest=sha256%3Ad1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17
2021/03/11 11:42:25 PUT /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:d1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:d1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
Pushing signature to: 127.0.0.1:63572/cosign-e2e:sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:a32319b5ad16e9cf1501850ca2fbc90f4dc23341123872bb8b1fe50f1dfc6226
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:d1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:34da2ce5fc5fe3687e044878ad16bed6ec32879ab91c57bf6f2ef7aa16f04169 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 HEAD /v2/cosign-e2e/blobs/sha256:bb561a72b2d685c6542f533b90967abd44dbc8855cea8ba34d8b9225821ce4c0 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/3510942875414458836
2021/03/11 11:42:25 PATCH /v2/cosign-e2e/blobs/uploads/2933568871211445515
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/3510942875414458836?digest=sha256%3A34da2ce5fc5fe3687e044878ad16bed6ec32879ab91c57bf6f2ef7aa16f04169
2021/03/11 11:42:25 PUT /v2/cosign-e2e/blobs/uploads/2933568871211445515?digest=sha256%3Abb561a72b2d685c6542f533b90967abd44dbc8855cea8ba34d8b9225821ce4c0
2021/03/11 11:42:25 PUT /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:d1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:34da2ce5fc5fe3687e044878ad16bed6ec32879ab91c57bf6f2ef7aa16f04169
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 GET /v2/cosign-e2e/manifests/sha256-698dc87529807a85dfb56ff24f01b6d0339ba22411638d3320c177513e92dba1.cosign
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:d1cba456fc9361b0270547b441b01341b5aeb3ad6d453855a2e3b17b455c9e17
2021/03/11 11:42:25 GET /v2/cosign-e2e/blobs/sha256:34da2ce5fc5fe3687e044878ad16bed6ec32879ab91c57bf6f2ef7aa16f04169
2021/03/11 11:42:25 GET /v2/
2021/03/11 11:42:25 DELETE /v2/cosign-e2e/manifests/latest
--- PASS: TestSignVerify (2.04s)
=== RUN   TestMultipleSignatures
    e2e_test.go:353: COSIGN_TEST_REPO unset, using fake registry
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:82d9a944e5031fb48f2e03c5f5a47d6e00490c12d3e1bf1b40cf36c349a90e80 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:b54afa1a7d838b27636406dcce16c814bd009795600306fa5fa3910b8fbf0662 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:5460e056d2528994126df8867a41e1c8286ad68e8957d66171f090ce481314fd 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:e0924ae83ac95679207f050d6be4d03512a42e530f7a022ac1fdfd054a794368 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:494bf9fe551d433782a9fe91eecc41d7e7a7187d18eac65e5c43da72dc7f499a 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:ffb45968bd6cbe48f73749b69c088ddfa0bdb729bab1e60f79938a6e1c0922e6 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/3328451335138149956
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/5263531936693774911
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/7955079406183515637
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/1874068156324778273
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/2703501726821866378
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/2740103009342231109
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/1874068156324778273?digest=sha256%3A82d9a944e5031fb48f2e03c5f5a47d6e00490c12d3e1bf1b40cf36c349a90e80
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/3328451335138149956?digest=sha256%3A494bf9fe551d433782a9fe91eecc41d7e7a7187d18eac65e5c43da72dc7f499a
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/5263531936693774911?digest=sha256%3Ab54afa1a7d838b27636406dcce16c814bd009795600306fa5fa3910b8fbf0662
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/2703501726821866378?digest=sha256%3A5460e056d2528994126df8867a41e1c8286ad68e8957d66171f090ce481314fd
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/7955079406183515637?digest=sha256%3Ae0924ae83ac95679207f050d6be4d03512a42e530f7a022ac1fdfd054a794368
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/2740103009342231109?digest=sha256%3Affb45968bd6cbe48f73749b69c088ddfa0bdb729bab1e60f79938a6e1c0922e6
2021/03/11 11:42:27 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
Pushing signature to: 127.0.0.1:63608/cosign-e2e:sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:8d8feba1a9ec01b2c11cc5c22b609d2dde9fc6dafec11bda873ffd0429d86db0 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/1905388747193831650
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/6941261091797652072
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/1905388747193831650?digest=sha256%3A37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/6941261091797652072?digest=sha256%3A8d8feba1a9ec01b2c11cc5c22b609d2dde9fc6dafec11bda873ffd0429d86db0
2021/03/11 11:42:27 PUT /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
Pushing signature to: 127.0.0.1:63608/cosign-e2e:sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:8d8feba1a9ec01b2c11cc5c22b609d2dde9fc6dafec11bda873ffd0429d86db0
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 HEAD /v2/cosign-e2e/blobs/sha256:96fde9c17fb0210fefb2cc794c895c2a43bac0e101e9c077ee19d7abde0e5dee 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:27 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:27 PATCH /v2/cosign-e2e/blobs/uploads/7981306761429961588
2021/03/11 11:42:27 PUT /v2/cosign-e2e/blobs/uploads/7981306761429961588?digest=sha256%3A96fde9c17fb0210fefb2cc794c895c2a43bac0e101e9c077ee19d7abde0e5dee
2021/03/11 11:42:27 PUT /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 GET /v2/cosign-e2e/manifests/sha256-73d60156e8dde8125f177739902a99b5162ee2cc55ebd1180071506c418397be.cosign
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 GET /v2/cosign-e2e/blobs/sha256:37229bb39ab2fab4f8bc1d6a7c7c739d80d3aa4d90eccc93e13c024b053ce06b
2021/03/11 11:42:27 GET /v2/
2021/03/11 11:42:27 DELETE /v2/cosign-e2e/manifests/latest
--- PASS: TestMultipleSignatures (1.81s)
=== RUN   TestGenerate
    e2e_test.go:353: COSIGN_TEST_REPO unset, using fake registry
2021/03/11 11:42:29 GET /v2/
2021/03/11 11:42:29 HEAD /v2/cosign-e2e/blobs/sha256:ed4143bf0109f4186b722e0ddff018a58aa6e784aaa7d7c4a93866cb08feea35 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:29 HEAD /v2/cosign-e2e/blobs/sha256:d48486ba0504bf13ed27ee0b7077e94ff9f270375eeaaedfc312e0518d1edc94 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:29 HEAD /v2/cosign-e2e/blobs/sha256:d90a5c65986c01aa64a59eb1ace438c52039ab98656cd187ceddb5e17a17d309 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:29 HEAD /v2/cosign-e2e/blobs/sha256:0741ee4529e95f4129183208cc43b3f645f9768f318a48cd4c56c726bbe28b36 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:29 HEAD /v2/cosign-e2e/blobs/sha256:15dcdc5f68c4301c058d0b2e0e8d665b4ddd8fbb94135f934c931a336c3fe126 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:29 HEAD /v2/cosign-e2e/blobs/sha256:49e0bf138c0096a41211454b8601ae662cb309ec121700bcc9ab0f60d8a864a1 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:29 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:29 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:29 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:29 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:29 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:29 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:29 PATCH /v2/cosign-e2e/blobs/uploads/8995016276575641803
2021/03/11 11:42:29 PATCH /v2/cosign-e2e/blobs/uploads/732830328053361739
2021/03/11 11:42:29 PATCH /v2/cosign-e2e/blobs/uploads/5486140987150761883
2021/03/11 11:42:29 PATCH /v2/cosign-e2e/blobs/uploads/6382800227808658932
2021/03/11 11:42:29 PATCH /v2/cosign-e2e/blobs/uploads/2781055864473387780
2021/03/11 11:42:29 PATCH /v2/cosign-e2e/blobs/uploads/545291762129038907
2021/03/11 11:42:29 PUT /v2/cosign-e2e/blobs/uploads/8995016276575641803?digest=sha256%3Aed4143bf0109f4186b722e0ddff018a58aa6e784aaa7d7c4a93866cb08feea35
2021/03/11 11:42:29 PUT /v2/cosign-e2e/blobs/uploads/732830328053361739?digest=sha256%3Ad48486ba0504bf13ed27ee0b7077e94ff9f270375eeaaedfc312e0518d1edc94
2021/03/11 11:42:29 PUT /v2/cosign-e2e/blobs/uploads/6382800227808658932?digest=sha256%3A15dcdc5f68c4301c058d0b2e0e8d665b4ddd8fbb94135f934c931a336c3fe126
2021/03/11 11:42:29 PUT /v2/cosign-e2e/blobs/uploads/2781055864473387780?digest=sha256%3A49e0bf138c0096a41211454b8601ae662cb309ec121700bcc9ab0f60d8a864a1
2021/03/11 11:42:29 PUT /v2/cosign-e2e/blobs/uploads/5486140987150761883?digest=sha256%3Ad90a5c65986c01aa64a59eb1ace438c52039ab98656cd187ceddb5e17a17d309
2021/03/11 11:42:29 PUT /v2/cosign-e2e/blobs/uploads/545291762129038907?digest=sha256%3A0741ee4529e95f4129183208cc43b3f645f9768f318a48cd4c56c726bbe28b36
2021/03/11 11:42:29 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:29 GET /v2/
2021/03/11 11:42:29 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:29 GET /v2/
2021/03/11 11:42:29 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:29 GET /v2/
2021/03/11 11:42:29 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:29 GET /v2/
2021/03/11 11:42:29 DELETE /v2/cosign-e2e/manifests/latest
--- PASS: TestGenerate (1.38s)
=== RUN   TestUploadDownload
    e2e_test.go:353: COSIGN_TEST_REPO unset, using fake registry
=== RUN   TestUploadDownload/file_containing_signature
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:ccc60dc456c1ecadb6900cb0b25a733798d57dc9be7ffa0e04643a0026fb5cbb 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:3ab161c3831768ecfee76fdb67065189c28701905c20d844a114b6996160890c 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:5062c0eacbfa77ac7aa40dd0a243f699394f61427478f9d05b752bf1c19b42e2 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:72156bec5ddbc687c8507f4b2e4f2cb9970d5bf556a08bc7075c07acd1a99a2f 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:34c003d38bbdb3473d0d99594925e38c82445da96e81e6db70b42f4ef02474da 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:a4cf8b0b87ee89013866f78697ed4436c4e443fe934118a1abd06dc2a1777c61 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/4893789450120281907
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/7273596521315663110
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/2338498362660772719
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/3337066551442961397
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/2601737961087659062
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/7273596521315663110?digest=sha256%3A5062c0eacbfa77ac7aa40dd0a243f699394f61427478f9d05b752bf1c19b42e2
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/8121576815539813105
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/4893789450120281907?digest=sha256%3A3ab161c3831768ecfee76fdb67065189c28701905c20d844a114b6996160890c
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/2338498362660772719?digest=sha256%3Accc60dc456c1ecadb6900cb0b25a733798d57dc9be7ffa0e04643a0026fb5cbb
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/3337066551442961397?digest=sha256%3A34c003d38bbdb3473d0d99594925e38c82445da96e81e6db70b42f4ef02474da
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/8121576815539813105?digest=sha256%3Aa4cf8b0b87ee89013866f78697ed4436c4e443fe934118a1abd06dc2a1777c61
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/2601737961087659062?digest=sha256%3A72156bec5ddbc687c8507f4b2e4f2cb9970d5bf556a08bc7075c07acd1a99a2f
2021/03/11 11:42:30 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 GET /v2/cosign-e2e/manifests/sha256-0828c76942faeeeb1f71cb6bd7f90a1e52c3a1c3a6ff15beea1edcd4c40f61a1.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:f5b1321b02e0f73b0e68ecaa6da3d1204ddc4341d62e896136fe69b75a86a1f6 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 HEAD /v2/cosign-e2e/blobs/sha256:17e25830de8833d4c96b15a69173bdbf69e66d620210daed42455095ddd73820 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/2740376916591569721
2021/03/11 11:42:30 PATCH /v2/cosign-e2e/blobs/uploads/8249030965139585917
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/2740376916591569721?digest=sha256%3Af5b1321b02e0f73b0e68ecaa6da3d1204ddc4341d62e896136fe69b75a86a1f6
2021/03/11 11:42:30 PUT /v2/cosign-e2e/blobs/uploads/8249030965139585917?digest=sha256%3A17e25830de8833d4c96b15a69173bdbf69e66d620210daed42455095ddd73820
2021/03/11 11:42:30 PUT /v2/cosign-e2e/manifests/sha256-0828c76942faeeeb1f71cb6bd7f90a1e52c3a1c3a6ff15beea1edcd4c40f61a1.cosign
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 GET /v2/cosign-e2e/manifests/sha256-0828c76942faeeeb1f71cb6bd7f90a1e52c3a1c3a6ff15beea1edcd4c40f61a1.cosign
2021/03/11 11:42:30 GET /v2/cosign-e2e/blobs/sha256:f5b1321b02e0f73b0e68ecaa6da3d1204ddc4341d62e896136fe69b75a86a1f6
2021/03/11 11:42:30 GET /v2/
2021/03/11 11:42:30 DELETE /v2/cosign-e2e/manifests/latest
=== RUN   TestUploadDownload/raw_signature_as_argument
2021/03/11 11:42:31 GET /v2/
2021/03/11 11:42:31 HEAD /v2/cosign-e2e/blobs/sha256:25e24e0af4222be205ca0ac3083cb13b7235dd3ac34f1acf0f3174d678d3ab2e 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:31 HEAD /v2/cosign-e2e/blobs/sha256:da1140b44621bf06dff5a225d2da17f82880d954919b2b189508b2b658a6580c 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:31 HEAD /v2/cosign-e2e/blobs/sha256:5c382987316a8ff64b19d2b07f0b2119d767ab87d2582ff231681146aefec39f 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:31 HEAD /v2/cosign-e2e/blobs/sha256:86d6ef17212b8a2e4371228018677cf8b5e2e2236d0982c48a1a0f395c7a9f6b 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:31 HEAD /v2/cosign-e2e/blobs/sha256:53d704931027a9d5fa06e9002378c00e295b79dda240a825fa857d261ca8dedc 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:31 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:31 HEAD /v2/cosign-e2e/blobs/sha256:f5b8c0d737729e5601aef80c3c56dbcf1c2c6ed28385ca721f6afb3d3f9a0583 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:31 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:31 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:31 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:31 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:31 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/8603989663476771718
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/6735196588112087610
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/6842348953158377901
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/2227583514184312746
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/8603989663476771718?digest=sha256%3A25e24e0af4222be205ca0ac3083cb13b7235dd3ac34f1acf0f3174d678d3ab2e
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/2873287401706343734
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/7388428680384065704
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/6735196588112087610?digest=sha256%3A53d704931027a9d5fa06e9002378c00e295b79dda240a825fa857d261ca8dedc
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/6842348953158377901?digest=sha256%3A86d6ef17212b8a2e4371228018677cf8b5e2e2236d0982c48a1a0f395c7a9f6b
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/7388428680384065704?digest=sha256%3Af5b8c0d737729e5601aef80c3c56dbcf1c2c6ed28385ca721f6afb3d3f9a0583
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/2227583514184312746?digest=sha256%3A5c382987316a8ff64b19d2b07f0b2119d767ab87d2582ff231681146aefec39f
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/2873287401706343734?digest=sha256%3Ada1140b44621bf06dff5a225d2da17f82880d954919b2b189508b2b658a6580c
2021/03/11 11:42:32 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/sha256-4d3dd9c9829d0a606652beed8118861f7e15f02f4ee65b48d050708c032d39ba.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:17e25830de8833d4c96b15a69173bdbf69e66d620210daed42455095ddd73820
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:f5b1321b02e0f73b0e68ecaa6da3d1204ddc4341d62e896136fe69b75a86a1f6
2021/03/11 11:42:32 PUT /v2/cosign-e2e/manifests/sha256-4d3dd9c9829d0a606652beed8118861f7e15f02f4ee65b48d050708c032d39ba.cosign
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/sha256-4d3dd9c9829d0a606652beed8118861f7e15f02f4ee65b48d050708c032d39ba.cosign
2021/03/11 11:42:32 GET /v2/cosign-e2e/blobs/sha256:f5b1321b02e0f73b0e68ecaa6da3d1204ddc4341d62e896136fe69b75a86a1f6
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 DELETE /v2/cosign-e2e/manifests/latest
=== RUN   TestUploadDownload/empty_signature_as_argument
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:ee6bb2cfc0dff05a5dfecb2b2ba5ebcd93a10a4c16a6f191561022f4e950d8a7 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:e1fae0070f1f6f0c2dd36852bd0e27ce958a495c184d1a901062bbb2bcb6ae0c 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:24fab67a665883245c19e62c2e143aa8259bba69e6d8a82beaf30a5700f895c2 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:c669f37751ee61d6dabef3f2512e7bb36e5ae61ef73be63602e550a64bb9d923 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:8997919a650f12dbdaed0fa91e23dc6570215cd65fd9e62944aa0d3b2845a985 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:32 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 HEAD /v2/cosign-e2e/blobs/sha256:0980cc3537ab2f91553fd1486b678f885d966cb0c58331f16ac6210df88e6427 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:32 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/8505906760983331750
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/837825985403119657
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/4548432111829895923
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/8549944162621642512
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/837825985403119657?digest=sha256%3A24fab67a665883245c19e62c2e143aa8259bba69e6d8a82beaf30a5700f895c2
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/8807817071862113702
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/4548432111829895923?digest=sha256%3Ae1fae0070f1f6f0c2dd36852bd0e27ce958a495c184d1a901062bbb2bcb6ae0c
2021/03/11 11:42:32 PATCH /v2/cosign-e2e/blobs/uploads/3209308858241334655
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/8505906760983331750?digest=sha256%3Aee6bb2cfc0dff05a5dfecb2b2ba5ebcd93a10a4c16a6f191561022f4e950d8a7
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/8549944162621642512?digest=sha256%3Ac669f37751ee61d6dabef3f2512e7bb36e5ae61ef73be63602e550a64bb9d923
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/8807817071862113702?digest=sha256%3A8997919a650f12dbdaed0fa91e23dc6570215cd65fd9e62944aa0d3b2845a985
2021/03/11 11:42:32 PUT /v2/cosign-e2e/blobs/uploads/3209308858241334655?digest=sha256%3A0980cc3537ab2f91553fd1486b678f885d966cb0c58331f16ac6210df88e6427
2021/03/11 11:42:32 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 GET /v2/cosign-e2e/manifests/sha256-af35eeefb0451634e103b6e7fd715ccf5d08b1080d59b501398ec5d4600f12cf.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:32 GET /v2/
2021/03/11 11:42:32 DELETE /v2/cosign-e2e/manifests/latest
--- PASS: TestUploadDownload (2.99s)
    --- PASS: TestUploadDownload/file_containing_signature (1.38s)
    --- PASS: TestUploadDownload/raw_signature_as_argument (0.79s)
    --- PASS: TestUploadDownload/empty_signature_as_argument (0.82s)
=== RUN   TestTlog
    e2e_test.go:353: COSIGN_TEST_REPO unset, using fake registry
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:72898f5cdc28404569c3f12fcc413c57c83713b6ebe347d18ee7c6ccf0f522ff 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:9b3e560963123ea784a4a63cc5df3774fd98ed3c861f106bbc3aab2a803a1fcd 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:05ca6173264e745e553dad75e3188420cb791576d3dafd903fad586dde4ef62c 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:55ca517a7c7aa8075154fe4edf84d0c7edb7729e47de545477f9cbf7863fae97 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:e3866990dbaf994d8bedab55f5a5ee4537199620314710d4e7eceefc17d87126 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:8d3b3c312110f3ae4094fb5b7aaaa9d204aedafaf3fbe9080289a18b99aa8181 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/6971241403795498694
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/3724427934598140041
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/2970700287221458280
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/1205043859388862788
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/8267293389953062911
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/1205043859388862788?digest=sha256%3Ae3866990dbaf994d8bedab55f5a5ee4537199620314710d4e7eceefc17d87126
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/9093919513921919021
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/6971241403795498694?digest=sha256%3A72898f5cdc28404569c3f12fcc413c57c83713b6ebe347d18ee7c6ccf0f522ff
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/2970700287221458280?digest=sha256%3A05ca6173264e745e553dad75e3188420cb791576d3dafd903fad586dde4ef62c
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/3724427934598140041?digest=sha256%3A9b3e560963123ea784a4a63cc5df3774fd98ed3c861f106bbc3aab2a803a1fcd
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/8267293389953062911?digest=sha256%3A55ca517a7c7aa8075154fe4edf84d0c7edb7729e47de545477f9cbf7863fae97
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/9093919513921919021?digest=sha256%3A8d3b3c312110f3ae4094fb5b7aaaa9d204aedafaf3fbe9080289a18b99aa8181
2021/03/11 11:42:33 PUT /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
Pushing signature to: 127.0.0.1:63718/cosign-e2e:sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign 404 MANIFEST_UNKNOWN Unknown manifest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:1c296162595e02ecec1d6574fb04c073176b199143feedcf8123ac0d072f2e76 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 HEAD /v2/cosign-e2e/blobs/sha256:982bc03e46d22e04936d33cdc173fdf1c0aae25f904e95b66b3c9d077a4553fc 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/5944830206637008055
2021/03/11 11:42:33 PATCH /v2/cosign-e2e/blobs/uploads/6651414131918424343
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/5944830206637008055?digest=sha256%3A982bc03e46d22e04936d33cdc173fdf1c0aae25f904e95b66b3c9d077a4553fc
2021/03/11 11:42:33 PUT /v2/cosign-e2e/blobs/uploads/6651414131918424343?digest=sha256%3A1c296162595e02ecec1d6574fb04c073176b199143feedcf8123ac0d072f2e76
2021/03/11 11:42:33 PUT /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:33 GET /v2/cosign-e2e/blobs/sha256:1c296162595e02ecec1d6574fb04c073176b199143feedcf8123ac0d072f2e76
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:33 GET /v2/cosign-e2e/blobs/sha256:1c296162595e02ecec1d6574fb04c073176b199143feedcf8123ac0d072f2e76
2021/03/11 11:42:33 GET /v2/
2021/03/11 11:42:33 GET /v2/cosign-e2e/manifests/latest
Pushing signature to: 127.0.0.1:63718/cosign-e2e:sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:34 GET /v2/
2021/03/11 11:42:34 GET /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:34 GET /v2/cosign-e2e/blobs/sha256:982bc03e46d22e04936d33cdc173fdf1c0aae25f904e95b66b3c9d077a4553fc
2021/03/11 11:42:34 GET /v2/
2021/03/11 11:42:34 HEAD /v2/cosign-e2e/blobs/sha256:1c296162595e02ecec1d6574fb04c073176b199143feedcf8123ac0d072f2e76
2021/03/11 11:42:34 HEAD /v2/cosign-e2e/blobs/sha256:03bde37aeb77179cd5ffb4e6f92308b61aaead7d2b35d455753184a05c59a825 404 BLOB_UNKNOWN Unknown blob
2021/03/11 11:42:34 POST /v2/cosign-e2e/blobs/uploads/
2021/03/11 11:42:34 PATCH /v2/cosign-e2e/blobs/uploads/788787457839692041
2021/03/11 11:42:34 PUT /v2/cosign-e2e/blobs/uploads/788787457839692041?digest=sha256%3A03bde37aeb77179cd5ffb4e6f92308b61aaead7d2b35d455753184a05c59a825
2021/03/11 11:42:34 PUT /v2/cosign-e2e/manifests/sha256-d80c967c6c29e51b980e8a0ac95405e83b26ea1b35455fa88a98576e782596de.cosign
2021/03/11 11:42:34 GET /v2/
2021/03/11 11:42:34 GET /v2/cosign-e2e/manifests/latest
    e2e_test.go:282: Post "http://127.0.0.1:3000/api/v1/log/entries": dial tcp 127.0.0.1:3000: connect: connection refused
2021/03/11 11:42:34 GET /v2/
2021/03/11 11:42:34 DELETE /v2/cosign-e2e/manifests/latest
--- FAIL: TestTlog (1.82s)
FAIL
FAIL	github.com/sigstore/cosign/test	10.534s
FAIL

</pre>
</details>

--- a/test/e2e_test.go
+++ b/test/e2e_test.go
@@ -1,3 +1,5 @@
+// +build e2e
+
 package test
 
 import (

--- a/test/e2e_test.sh
+++ b/test/e2e_test.sh
@@ -29,7 +29,7 @@ echo "running tests"
 
 cd $GITHUB_WORKSPACE
 go build -o cosign ./cmd/cosign
-go test ./...
+go test -tags=e2e ./...
 
 
 echo "cleanup"

